apps/blockchain/lib/blockchain/interface/account_interface/cache.ex

```diff
@@ -0,0 +1,71 @@
+defmodule Blockchain.Interface.AccountInterface.Cache do
+  alias Blockchain.Account.Address
+  defstruct cache: %{}
+
+  @type key_cache() :: %{
+          integer() => %{current_value: integer() | :deleted, initial_value: integer() | nil}
```

Comment:
while working on https://github.com/poanetwork/mana/issues/416, I will try using `ets` tables as cache storage instead of simple maps